src/main/java/com/junwoo/ott/domain/auth/dto/request/AuthLoginRequestDto.java

```diff
@@ -0,0 +1,22 @@
+package com.junwoo.ott.domain.auth.dto.request;
+
+import com.junwoo.ott.domain.auth.dto.body.AuthLoginDto;
+import jakarta.validation.Valid;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+
+@Getter
+@NoArgsConstructor
+@AllArgsConstructor
+public class AuthLoginRequestDto {
+
+  private String username;
+  private String password;
+
+  public AuthLoginRequestDto(@Valid AuthLoginDto authLoginDto) {
```

Comment:
@Validated 어노테이션 사용을 권장합니다.
```suggestion
  public AuthLoginRequestDto(@Validated AuthLoginDto authLoginDto) {
```